### DISCUSSION TITLE
I’m not alive, but I can grow; I don’t have lungs, but I need air; I don’t have a mouth, but water kills me. What am I?

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#10] [Galaxy Trivia] [answered]
Puzzle: I’m not alive, but I can grow; I don’t have lungs, but I need air; I don’t have a mouth, but water kills me. What am I?

**Comment by hey-viral** [comment, marked as answer]
Fire 🔥

**Reply by meviru** [reply]
Congratulations!! 👏👏  You've just earned the [Galaxy Brain](https://github.com/hey-viral?tab=achievements&achievement=galaxy-brain) badge. Keep it up, and remember to give the repository a star to show your support! 🎉

**Comment by srijon57** [comment]
Fire 🔥

**Reply by meviru** [reply]
@srijon57 This question has already been answered. Please check other questions.

**Comment by tusharmishra069** [comment]
Fire

**Reply by meviru** [reply]
@tusharmishra069  This question has already been answered. Please check other questions.

**Comment by shynekomaid** [comment]
Also rust. 

- “Not alive”: Rust isn't a living organism, but it can spread across metal surfaces.
- “Can grow”: Rust expands and spreads over time, especially with exposure to air and moisture.
- “Needs air”: Oxygen is necessary for the oxidation process that creates rust.
- “Water kills me”: While water accelerates rusting, too much water can saturate the surface, slowing or preventing further rust development.

**Reply by meviru** [reply]
- Rust is a chemical process that happens when iron or steel reacts with moisture and oxygen.
- Rust doesn’t grow in the sense implied by the puzzle; it forms as a result of oxidation.
- Rust does not need air in the same way fire does; it requires moisture (water) and oxygen.
- Rust cannot be killed by water; in fact, water is one of the key components in the formation of rust.

**Reply by meviru** [reply]
You can check out our other riddles and puzzles in the discussion section.

**Comment by PranjalTrivedi** [comment]
I am Claiming Galaxy Brain Badge
